pgxn/neon/pagestore_client.h

```diff
@@ -62,7 +62,7 @@ typedef struct
 typedef struct
 {
 	NeonMessageTag tag;
-	bool		latest;			/* if true, request latest page version */
+	XLogRecPtr	horizon;		/* uppe boundary for page LSN */
```

Comment:
I have removed comment because I do not consider 0 as special case any more.
`get_page` now specifies interval of LSN. 0 is valid value of interval low boundary.
It is not somehow specially treated at PS.